client-app/src/examples/todo/TodoPanel.js

```diff
@@ -0,0 +1,72 @@
+import {grid} from '@xh/hoist/cmp/grid';
+import {filler, fragment} from '@xh/hoist/cmp/layout';
+import {creates, hoistCmp} from '@xh/hoist/core';
+import {panel} from '@xh/hoist/desktop/cmp/panel';
+import {toolbar, toolbarSep} from '@xh/hoist/desktop/cmp/toolbar';
+import {taskDialog} from './TaskDialog';
+import {TodoPanelModel} from './TodoPanelModel';
+import {switchInput} from '@xh/hoist/desktop/cmp/input';
+import {recordActionBar} from '@xh/hoist/desktop/cmp/record';
+
+
+export const todoPanel = hoistCmp.factory({
+    model: creates(TodoPanelModel),
+
+    render({model}) {
+        return fragment(
+            taskDialog(),
+            panel({
+                tbar: tbar(),
+                item: grid({onRowDoubleClicked: (e) => model.taskDialogModel.openEditForm(e.data.data)}),
```

Comment:
I like this form of openEditForm() better -- where we pass the task, rather than get from selection.  But need to actually change the implementation, which is ignoring this argument and getting from selection.  Note that we need to also account for the fact that group rows can be clicked and nothing should happen in that case.